workshop-pydata-part1.ipynb

```diff
@@ -225,7 +225,7 @@
    "outputs": [],
    "source": [
     "weather_data_batch.expect_column_max_to_be_between(\n",
-    "    column=\"lon\", min_value=-71, max_value=6, \n",
+    "    column=\"lon\", min_value=-74, max_value=-34, \n",
```

Comment:
more specifically; this :)

Reply:
This was a little bit random from my memory. I changed this already in the final update of the notebook, could you discard edits to the notebook in this PR to avoid merge conflicts? :)